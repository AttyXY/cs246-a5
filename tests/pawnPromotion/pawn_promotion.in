game human human
move g2 g4
move h7 h5
move g4 h5
move g7 g6
move h5 h6
move g6 g5
move h6 h7
move g5 g4
move h7 g8 N
quit
quit
