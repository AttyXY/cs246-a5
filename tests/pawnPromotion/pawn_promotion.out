WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING GAME!

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  --------
3  --------
2  PPPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  ------P-
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppppppp-
6  --------
5  -------p
4  ------P-
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppppppp-
6  --------
5  -------P
4  --------
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppp--
6  ------p-
5  -------P
4  --------
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppp--
6  ------pP
5  --------
4  --------
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppp--
6  -------P
5  ------p-
4  --------
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppp-P
6  --------
5  ------p-
4  --------
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppp-P
6  --------
5  --------
4  ------p-
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

8  rnbqkbNr
7  pppppp--
6  --------
5  --------
4  ------p-
3  --------
2  PPPPPP-P
1  RNBQKBNR
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
